### PR TITLE
c++11 style to disable copy constructor

### PR DESCRIPTION
**Pull request by 1vanK**

### CODE REVIEW

**Comment by 1vanK** [comment]
Is URHO3D_DOCS currently enabled by default?

**Comment by weitjong** [comment]
Thanks for your PR. We will merge it shortly.

The "URHO3D_DOCS" build option is not enabled by default. See the table in https://urho3d.github.io/documentation/HEAD/_building.html#Build_Options.

**Comment by 1vanK** [comment]
d:\MyGames\Urho3DCurrent\Build>cmake.exe ../Urho3D -G "Visual Studio 15" -DURHO3D_OPENGL=1 -DURHO3D_ANGELSCRIPT=1 -DURHO3D_LUA=0 -DURHO3D_STATIC_RUNTIME=1 -DURHO3D_SAMPLES=1 -DURHO3D_TOOLS=0

```
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
-- Configuring done
-- Generating done
-- Build files have been written to: D:/MyGames/Urho3DCurrent/Build
```

**Comment by weitjong** [comment]
@1vanK Let me try again. I think I understand what your concern is now. The "doc" built-in target is not an option. The build system wants to generate it a long with other built-in targets: "clean", "install", etc. Thus, the build system always tries to find the Doxygen and complain about it when it is not found. The "URHO3D_DOCS" is an option to tell build system to auto invoke the "doc" target as part of the "all" build or not. The "doc" target is there regardless. I hope this clarifies.

**Comment by 1vanK** [comment]
Thanks for clarifying
